strings/src/main/java/com/revenuecat/purchases/strings/ConfigureStrings.kt

```diff
@@ -11,4 +11,18 @@ object ConfigureStrings {
             "Make sure you configure Purchases before trying to get the default instance. " +
             "More info here: https://errors.rev.cat/configuring-sdk"
     const val SDK_VERSION = "SDK Version - %s"
+    const val LEGACY_API_KEY = "Looks like you're using a legacy API key.\n" +
+        "This is still supported, but it's recommended to migrate to using platform-specific API key, " +
+        "which should look like 'goog_1a2b3c4d5e6f7h' or 'amzn_1a2b3c4d5e6f7h'.\n" +
+        "See https://rev.cat/auth for more details."
+    const val AMAZON_API_KEY_GOOGLE_STORE = "Looks like you're using an Amazon API key but have configured the SDK " +
+        "for the Google play store.\nEither use a Google API key which should look like 'goog_1a2b3c4d5e6f7h' or " +
+        "configure the SDK to use Amazon.\nSee https://rev.cat/auth for more details."
+    const val GOOGLE_API_KEY_AMAZON_STORE = "Looks like you're using a Google API key but have configured the SDK " +
+        "for the Amazon app store.\nEither use an Amazon API key which should look like 'amzn_1a2b3c4d5e6f7h' or " +
+        "configure the SDK to use Google.\nSee https://rev.cat/auth for more details."
+    const val INVALID_API_KEY = "The specified API Key is not recognized.\n" +
+        "Ensure that you are using the public app-specific API key, " +
+        " which should look like 'goog_1a2b3c4d5e6f7h' or 'amzn_1a2b3c4d5e6f7h'.\n" +
+        "See https://rev.cat/auth for more details."
```

Comment:
good touch on the `\n`s

Reply:
In all honesty, I copied the message from iOS, but yeah, it improves readability I would say 👍